### DISCUSSION TITLE
Navbar Tailwind con flowbite

### DISCUSSION THREAD

**Discussion by jvaneb** [themesberg/flowbite-angular#49] [Q&A]
Hola comunidad

Estoy intentando crear un navbar que sea responsivo, pero cuando cambio a una pantalla menor, ejemplo vista desde celular no se desglosa el dropdown del menú al darle clic.

Quisiera saber si alguien tiene un ejemplo de como realizar esto o como integrar flowbite para que funcione correctamente en angular.

Aclaro que apenas estoy conociendo Tailwind no lo conozco, también integré flowbite en Angular debido a recomendaciones que vi, o si saben que otra librería se integra bien para esto.

![dependencias de proyecto](https://github.com/user-attachments/assets/b136885f-8852-4efb-bd13-0cc2ef20e4ce)

Muchas gracias por la ayuda.

**Comment by MGREMY** [comment]
Hola @jvaneb,

Sorry I don't speak spanish
This library has been "paused" for quite a long time, but opefully it will have a new release soon (let me create the e documentation and the release will be out)
I could suggest you to wait until the new version of flowbite-angular, because flowbite components doesn't fit well with angular, so we need to re-create components from scratch :)
